### PR TITLE
Added quickstart readme

### PR DESCRIPTION
**Pull request by natewalck**
Readme to get started with our chef cookbooks.

### CODE REVIEW

**Comment by mikedodge04** [comment]
`
# Building the Json file for the chef zero run
{
    'minimal_ohai' => true,
    'run_list' => ['recipe[cpe_init]']
}
`

**Comment by natewalck** [comment]
Tested and working.

File: chef/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    # Add a launchd that echos nothing
+    node.default['cpe_launchd']['com.MYCOMPANY.chef.CPE.doesnothing'] = {
+      'program_arguments' => ['echo', 'nothing'],
+      'run_at_load' => true,
+    }
+
+In terminal, cd to /Users/Shared/IT-CPE/chef/cookbooks, do a local chef run:
+
+    cd /Users/Shared/IT-CPE/chef/cookbooks
+    sudo chef-client -z -r "recipe[cpe_init]"
```
Comment (mikedodge04) [inline chef/README.md]: This should be 
`sudo chef-client -z -j quickstart.json`

**Comment by natewalck** [comment]
Should be good now.

**Pull request merged**